processing/upload_checks.py

```diff
@@ -67,6 +67,7 @@ def upload_checks(
             return None, False, None
     else:
         process_id = 999999
+        start_time = dt.strptime('2024-03-25 09:00', '%Y-%m-%d %H:%M')
```

Comment:
If you are using amf_utils, and there is a suitable format to use, might be a good idea to have that instead of a hardcoded string for the format.

Reply:
We are not using amp_utils b/c that is not publicly available at the moment. But changed to standard format in utils.